### DISCUSSION TITLE
Added: LDM notes for July 10th and July 17th

### DISCUSSION THREAD

**Discussion by agocke** [dotnet/csharplang#2702] [General]
[July 10th](https://github.com/dotnet/csharplang/blob/master/meetings/2019/LDM-2019-07-10.md)

1. Empty switch statement
1. `[DoesNotReturn]` attribute
1. Revisiting the `param!` null-checking feature

[July 17th](https://github.com/dotnet/csharplang/blob/master/meetings/2019/LDM-2019-07-17.md)

1. [Nullability of events](https://github.com/dotnet/roslyn/issues/34982)

1.  Triage, see https://github.com/dotnet/csharplang/projects/4
    - Triage championed features https://github.com/dotnet/csharplang/projects/4#column-4649189
    - Triage milestones

**Comment by svick** [comment]
> No place to put `!` on the property setter

What about `set!`? E.g.:

```c#
public string P { get; set!; } // auto-property

public string P // regular property
{
    get => …;
    set! => …;
}
```
